girder/api/describe.py

```diff
@@ -465,10 +464,10 @@ def listResources(self, params):
 
         routeMap = _apiRouteMap()
 
-        for resource in sorted(six.viewkeys(docs.routes), key=str):
+        for resource in sorted(docs.routes.keys(), key=str):
```

Comment:
For a dict `x`, `in x` is the same as `in x.keys()` so we could get rid of a lot of the calls to `.keys()`, but I don't really care either way.

Reply:
Agreed. However, I tried to transform the rest of the code minimally with these changes, so that would be a little out of scope.